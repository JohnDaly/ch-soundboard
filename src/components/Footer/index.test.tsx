// External Dependencies
import { mount } from 'enzyme'
import enzymeToJSON from 'enzyme-to-json'
import 'jest-styled-components'
import React from 'react'

// Internal Dependencies
import { Footer } from './index'

describe('Footer', () => {
    it('should match the snapshot', () => {
        const wrapper = mount(
            <Footer>
                <div>Footer Content</div>
            </Footer>
        )
        expect(enzymeToJSON(wrapper)).toMatchSnapshot()
    })
})
