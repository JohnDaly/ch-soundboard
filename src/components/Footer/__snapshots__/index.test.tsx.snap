// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match the snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: end;
  -webkit-justify-content: end;
  -ms-flex-pack: end;
  justify-content: end;
  position: absolute;
  bottom: 0;
  width: 100%;
}

<Footer>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "lastClassName": "c0",
          "rules": Array [
            "
          display: flex;
          flex-direction: row;
          justify-content: end;
          position: absolute;
          bottom: 0;
          width: 100%;
      ",
          ],
        },
        "displayName": "Footer",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "footer",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <footer
      className="c0"
    >
      <div>
        Footer Content
      </div>
    </footer>
  </StyledComponent>
</Footer>
`;
